docs/content/getting-started/troubleshooting.md

```diff
@@ -76,7 +76,7 @@ The configuration we use for wgpu can be influenced in the following ways:
     Naturally, support depends on your OS. The default backend is `vulkan` everywhere except on Mac where we use `metal`.
     On the web we prefer WebGPU and fall back automatically to WebGL if no support for WebGPU was detected.
     -   For instance, you can try `rerun --renderer=gl` or for the web viewer respectively `rerun --web-viewer --renderer=webgl`.
-    -   Alternatively, for the native viewer you can also use the `WGPU_BACKEND` environment variable with the above values.
+    -   Alternatively, for the native Viewer you can also use the `WGPU_BACKEND` environment variable with the above values.
```

Comment:
Some of these capitalised uses do feel weird—not only here specifically, but that's a good example Not sure I can make an actionable suggestion though. Maybe we should accept "viewer" and just enforce "Rerun Viewer"?